assets/js/messages.json

```
@@ -224,15 +224,9 @@
         ],
         [
             {
-                "project": ["mc", "mcd", "mcpe", "mcl", "mcapi", "mce"],
-                "name": "Report not in English",
-                "message": "*Thank you for your report!*\nHowever, this issue is {color:#FF5722}*Invalid*{color}.\n\nBug reports are only accepted in English, sorry.\nFeel free to create a new report if you can speak English.\n\n%quick_links%",
-                "fillname": []
-            },
-            {
-                "project": ["bds"],
+                "project": ["bds", "mc", "mcd", "mcpe", "mcl", "mcapi", "mce"],
                 "name": "Report not in English",
-                "message": "*Thank you for your report!*\n\nUnfortunately we are only able to accept bug reports in English. You are welcome to use an online translator, if that helps. (You can find an example bug ticket [here|https://help.minecraft.net/hc/en-us/articles/360039268071].)\n\nAlso, we'd recommend making use of the search feature before writing a new bug report, as it's very likely that the issue has been mentioned already, and you can add a vote and any new information to the main ticket.\n\n%quick_links%",
+                "message": "*Thank you for your report!*\nThank you for taking the time to report a bug. Unfortunately we are only able to accept bug reports in English since it is the common language used on this tracker.\n\nYou are welcome to create a new ticket about your issue in English. You can use an online translator, if it helps.\nHowever, we recommend making use of the [*+search feature+*|https://bugs.mojang.com/issues/] before writing a new bug report, as it's very likely that someone else already reported your issue before.\nIf you find an existing ticket about your issue, you can add a vote and any new information to it instead of reporting it yourself.\n\nPlease note that this bug tracker is not for support. If you have a payment issue or an issue with your account, please use the *Customer Support* link below.\n\n%quick_links%",
```

```suggestion
                "message": "*Thank you for your report!*\nThank you for taking the time to report a bug. Unfortunately we are only able to accept bug reports in English since it is the common language used on this tracker.\n\nYou are welcome to create a new ticket about your issue in English. You can use an online translator, if it helps.\nHowever, we recommend making use of the [*+search feature+*|https://bugs.mojang.com/issues/] before writing a new bug report, as it's very likely that someone else already reported your issue before.\nIf you find an existing ticket about your issue, you can add a vote and any new information to it instead of reporting it yourself.\n\nPlease note that this bug tracker is not for support. If you have a payment issue or an issue with your account, please use the *[Customer Support|https://help.minecraft.net/hc/en-us/requests/new]*.\n\n%quick_links%",
```
See #18, we no longer refer to the quick links, but include the link directly to avoid confusion.

---

It's a bit confusing to have "Community Support" and "Customer Support" as they look quite a lot alike, and could be easily misunderstood. How about "Mojang Customer Support" and "Community Support Discord". Also the text is getting quite long. Could shorten/adjust the part about duplicates to say:
"Please *[search|...] for duplicates* before creating a new ticket, as it is very likely that someone has reported your issue before. If you find an existing ticket about your issue, add a vote and any new information instead of reporting it again. Duplicate tickets are simply closed by the moderators and will not be forwarded to the developers."

---

Did you mean to reply to https://github.com/mojira/helper-messages/pull/37#discussion_r416609971?

---

Yep. The github commenting hierarchy sometimes confuses me.

---

@MikeUnverferth As for "Community Support" vs "Customer Support", that's universal for all quick links. Perhaps it would make sense to raise a new issue with this. I've changed it for this PR now.

As for the duplicates phrasing, I've shortened it a bit according to your suggestion, but it's not really a large difference. I don't think we need to include "Duplicate tickets are simply closed by the moderators and will not be forwarded to the developers". Perhaps that should be added to the Duplicate message itself though.